### ISSUE TITLE
AM/PM option

### ISSUE THREAD

**Issue by Dasonk** [ThinkRstat/prompt#2]
I thought it might be nice (and easy enough to implement) to have an option for the time stamp to be in am/pm format instead of 24-hour format.  However if implemented it might make sense to convert the shortcut for the 24-hour format to `T` instead of `t` since when formatting time "%T" is equivalent to "%H:%M:%S".

If that seems reasonable I could throw together a PR.

**Comment by romainfrancois** [comment]
I've been loosely following this https://www.cyberciti.biz/tips/howto-linux-unix-bash-shell-setup-prompt.html

So I guess you're right, here is the relevant extract: 

`\t` : the current time in 24-hour HH:MM:SS format
`\T` : the current time in 12-hour HH:MM:SS format
`\@` : the current time in 12-hour am/pm format
`\A` : the current time in 24-hour HH:MM format

**Comment by Dasonk** [comment]
Oh ok. Then leaving it as t makes sense. I'll try to get together a PR for some of those if you'd be interested in that.

**Comment by romainfrancois** [comment]
sure. Thanks :-)

**Comment by romainfrancois** [comment]
@Dasonk did you ever work on a PR ?

**Comment by Dasonk** [comment]
No. I had good intentions but life got in the way and I kind of forgot about it.
